src/App.jsx

```diff
@@ -14,70 +16,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [title, setTitle] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const handleTitleChange = (good) => {
+    if (good === title
+      || !good
+      || typeof good !== 'string') {
+      setTitle('');
```

Comment:
[NON BLOCKER] I think this is a little overkill

Reply:
I changed `''` to `null`, I think it is more clear. Also, I prefer to define one function to make changes in the title instead of two different ones, as I made in the first commit.